### ISSUE TITLE
Has this been given up on??

### ISSUE THREAD

**Issue by Ainaemaet** [volotat/SD-CN-Animation#173]
Hello, no updates in months and people have been asking for just as long about the twisting/warping issue that has made this otherwise **amazing** extension all but useless for text-to-video.

Not trying to be rude as I appreciate everything that any open source developer brings to us, but why isn't the dev addressing any of this?

It would be a real shame for this one to fall by the wayside - I only hope someone else picks up the mantle and keeps this going in it's been given up on!

**Comment by Johnnyprim3** [comment]
> Hello, no updates in months and people have been asking for just as long about the twisting/warping issue that has made this otherwise **amazing** extension all but useless for text-to-video.
> 
> Not trying to be rude as I appreciate everything that any open source developer brings to us, but why isn't the dev addressing any of this?
> 
> It would be a real shame for this one to fall by the wayside - I only hope someone else picks up the mantle and keeps this going in it's been given up on!

I feel EXACTLY the same way. The potential is astounding but unfortunately it seems to have been neglected. There simply is NO OTHER TEXT2VIDEO THAT COMPARES! I just hope the Author is fine, health wise. Cheers brother

**Comment by Ainaemaet** [comment]
Indeed, I hope so as well - part of the reason I wanted to ask actually.

If you're the author of this repo and you're reading this, think you could give us all a shout and let us know things are ok?

Best wishes and great health to everyone here. 🙂
________________________________

**Comment by rickylars** [comment]
I feel the same way, it's an amazing extension and I wish it would keep developing.

**Comment by peteTater** [comment]
A big collective THANK YOU is what the dev deserves from us all for sharing this creation. Let's be patient, grateful and compassionate. Surely some of you will remember how impatient and downright nasty some folk got during Auto-gate. It was only a month Auto took off,  and the whiners and haters surfaced fast. Sure we got Vlad stepping up and offering many benefits with SD.Next, but the attitude of some in the community was really disconcerting. Not saying anyone hasn't been courteous on this thread so far, of course.

Perhaps there's been a face melting heatwave in Armenia, a new job offer came up, a family bereavement, or the dude just had a burnout while going maximal on the batching stuff while the Issues, gripes grumbles and groans continued to flood in. A bunch of strangers asking for more more more (I include myself in that category!) can be pretty draining when you're building something for the love of it. Maybe there's big plans for 1.0 beyond any of our hopes and dreams, and anyways he's not disappeared altogether. He chimed in on the share your anim's thread within the last couple weeks and I saw a few tweets as well, so let's all just keep the hope and gratitude alive.

If you're having unsatisfactory results with txt2vid presently, for now, may I suggest that rolling back is the way to go (search PURPIFY in Issues for a guide from volotat if you haven't had to do a git rollback before, it's super easy). I have multiple versions running on several SD instances and getting distinctly different qualities from various commits. There may be some compatibilty factors with Gradio versions or other plugins causing these distinctive qualities in my own case, but after some serious initial faffing I'm now getting the benefit from different commits, whatever the cause, so it's well worth a try. That said, the newest commit has a lot to offer _thanks_ to the twisty warps if you're open to working with those qualities to your advantage. Will try to post some vids on the show your anims discussion. I'm getting beautiful and largely coherent results. 

@volotat I'll be teaching an AI Masterclass later today and believe me I'll be plugging SD-CN Animation to all the students. Between this and Deforum, there's so much fun to be had :D Hopefully some of the students will get hooked and I'll be encouraging them to drop by, leave a star and a thanks, along with sharing their work on the anim discussions thread. 

Cheers to the big chap, and hope you're good!

**Comment by peteTater** [comment]
@Ainaemaet Completely fair. My own concern, having been one of the folks getting perhaps overexcited a couple months back about all the talk of batching, fanatical as I am about the tool, was that if the dev's hit a stumbling block, it might be worth offering some encouragement and understanding in case others feeling frustrated with the wait might begin to complain or discuss a mutiny. There have been some unconstructive comments on other threads over the last couple months. Hope I didn't cause any offence to those who have also been offering words of concern and support.

**Comment by Ainaemaet** [comment]
I admire your thoughtfulness, and certainly don't agree with those who
would belittle or berate any developer, let alone one offering a free
product; though the longer one waits without asking questions the longer
one goes without answers - even if the answer is silence.

At this point I think it's reasonable to assume that if life isn't out of
the way it's in it - again to hoping all is well.

**Comment by kzhjymug** [comment]
Agree I want to see it continue too, I have not found anything comparable. I'm glad to see at least some people here recognize that the original creator has a life, and trying to track breaking and chaotic random and undocumented changes in A1111 is a huge task. I would not have had the dedication to take it as far as he did, so hats off to him.

I wonder if anyone else here has much familiarity with how all the moving parts work with A1111? If someone else has a fix, please fork it, fix it, and let us know. I am sure if OP ever comes back he would be happy to see there's a PR to fix the issues. Otherwise we will manage.

If you're reading this, thanks volotat, you have given us a lot and asked us for nothing in return; you are awesome.
